moveit_core/utils/src/robot_model_test_utils.cpp

```diff
@@ -34,7 +34,6 @@
 
 /* Author: Bryce Willey */
 
-#include <boost/algorithm/string_regex.hpp>
 #include <boost/math/constants/constants.hpp>
```

Comment:
You could get rid of this because PI is available from `math.h`. I don't care too much about this suggestion, though

Reply:
Should I use `#define _USE_MATH_DEFINES` before including `cmath` or `math.h`, or use `add_definitions(-D_USE_MATH_DEFINES)`? Or should we just leave it as using boost? We might be able to remove boost from some `CMakeList.txt`s or `package.xml`s if we change.

Reply:
If this stackexchange answer is right, you can just do `#include <math.h>`

https://stackoverflow.com/a/1727896

No need to #define anything or add_definitions as far as I can tell since we're targeting newer systems. I could be wrong, though.

Reply:
Ok great sounds good, I'll switch to that then.